ocs_ci/utility/vsphere.py

```diff
@@ -543,3 +543,44 @@ def get_used_unit_number(self, vm):
             device.unitNumber for device in vm.config.hardware.device
             if hasattr(device.backing, 'fileName') and device.unitNumber != 0
         ]
+
+    def is_folder_exists(self, name, cluster, dc):
```

Comment:
I would rename this to `folder_exists` or `check_folder_exists`

Reply:
Done